response_operations_social_ui/templates/social-view-case-details.html

```diff
@@ -1,74 +1,76 @@
-        </dd>
-      </dl>
-      <section class="social-case-section u-mt-l">
-        <h2 class="neptune">Property address</h2>
-        <dl class="survey-info info-panel" id="case-attributes">
-          {% for name in displayed_attributes %}
-            <dt class="case-address__{{ name|lower }}" id="case-{{ name|lower|replace('_', '-') }}">
-              {{ attributes[name] }}
-            </dt>
-          {% endfor %}
-        </dl>
-      </section>
-      <section class="social-case-section u-mt-xl">
-        <h2 class="neptune">Unique access code</h2>
-        <p class="mars" id="new-iac-count">
-          There have been {{iac_count}} unique access codes issued for this case.
-        </p>
-        <form action="{{ url_for('social_bp.generate_iac') }}" method='POST'>
-          <input type="hidden" name="case_id" value="{{case_id}}"/>
-          <button type="submit" class="btn btn--secondary btn--small generate-iac-submit" id="generate-iac-submit">Generate new code</button>
-        </form>
-        {% if new_iac %}
-          <dl class="survey-info info-panel" id="new-iac-iac">
-            <dt class="saturn u-mt-s u-mb-s" id="new-iac">
-              {{new_iac}}
-            </dt>
-          </dl>
-        {% endif %}
-      </section>
     </div>
+  {% endif %}
+  {% for msg in get_flashed_messages(category_filter=["success"]) %}
+    <div class="panel panel--simple panel--success u-mt-l">
+      <div id="status_change_successful" class="panel__body">
+        <h2 class="venus">{{ msg }}</h2>
+      </div>
+    </div>
+  {% endfor %}
+  <h1 class="jupiter u-mt-m" id="case_ref">{{case_reference}}</h1>
+  <dl class="metadata metadata__list grid grid--gutterless" title="Item metadata" aria-label="Item metadata">
+    <dt class="metadata__term grid__col col-1@m venus">Status:</dt>
+    <dd class="metadata__value grid__col col-11@m venus">
+    <span class="response-status-headline" id="case-group-status">
+      {{status}}
+    </span>
+    <span class="response-status-detail mars u-mt-xs u-db">
+      {{ updated_status }}
+    </span>
+    {% if case_event_description is not none %}
+    <span class="response-status-detail mars u-mt-xs u-db" id="case-group-status-detailed">
+      {{ case_event_description }}
+    </span>
+    {% endif %}
+    {% if can_change_status %}
+    <div class="response-status-action u-mt-s u-db">
+      <form action="{{ url_for('social_bp.change_case_response_status', case_id=case_id) }}" method='GET'>
+        <button type="submit" class="btn btn--secondary btn--small" id="change-status-submit">Change Status</button>
+      </form>
+    </div>
+    {% endif %}
+  </dd>
+  </dl>
+  <div class="case-details u-mt-m">
+    <section class="social-case-section u-mt-l">
+      <h2 class="neptune">Property address</h2>
+      <div class="address" id="case-attributes">
+        {% for name in displayed_attributes %}
+          <div class="case-address__{{ name|lower }}" id="case-{{ name|lower|replace('_', '-') }}">
```

Comment:
The changes here are making the address all caps. If this is intentional, the acceptance tests will need updating before this goes in.

Reply:
There's no written guidance on using the address component, so I'll check with eQ.  Should the acceptance tests be case-sensitive?

Reply:
@AdamHawtin  It turns out the pattern needs to be deprecated - so I'll amend this.

Reply:
The acceptance tests just string match on the data it was set up with. I believe in the past our philosophy was always display the address as it comes to us/is stored.

Reply:
@AdamHawtin Can you check the amendments?